apps/data-backend/src/index.ts

```diff
@@ -1,39 +1,33 @@
 import express from "express";
-import {server} from "./graphql"
 import dotenv from "dotenv";
 import router from "./router";
-import helmet from "helmet"
+import helmet from "helmet";
 import { launchBot, sendWebAppButton } from "./services/telegram-app";
-const cors = require("cors")
+const cors = require("cors");
+
 dotenv.config();
 const app = express();
-app.use(cors())
-app.use(helmet())
-// Alternatively, you can configure CORS for specific routes or origins
+
+app.use(helmet());
 app.use(cors({
-  origin: 'http://localhost:8081' // Adjust this to match the URL of your React Native web version if necessary
+    origin: 'http://localhost:8081'
```

Comment:
Can we add this with the .env APP_URL_WEB please? IF you can add this env in the .env.example too please

Reply:
Noted